jekyll/_cci2/authorize-google-cloud-sdk.md

```diff
@@ -4,62 +4,66 @@ title: Authorize Google Cloud SDK
 description: How to authorize the Google Cloud SDK
 categories: [deploying]
 redirect_from: /google-auth
-contentTags: 
+contentTags:
```

Comment:
There are a couple of **notes** on this page that need formatting as banners, GH won't let me comment on those lines though